docs/NOTICES.md

```diff
@@ -928,17 +930,75 @@
 * [`stops.txt`](http://gtfs.org/reference/static#stopstxt)
 * [`fare_rules.txt`](http://gtfs.org/reference/static#farerulestxt)
 
-#### [TooFastTravelNotice](/RULES.md#TooFastTravelNotice)
+#### [FastTravelBetweenConsecutiveStopsNotice](/RULES.md#FastTravelBetweenConsecutiveStopsNotice)
+##### Fields description
+
+| Field name        	| Description                             | Type   	|
+|-----------------------|---------------------------------------- |--------	|
+| `tripCsvRowNumber`    | The row number of the problematic trip. | Long 	|
+| `tripId`           	| `trip_id` of the problematic trip.      | String 	|
+| `routeId`             | `route_id` of the problematic trip.     | String 	|
+| `speedKph`         	| Travel speed (km/h).                    | Double 	|
+| `distanceKm`       	| Distance between stops (km).            | Double 	|
+| `csvRowNumber1`       | The row number of the first stop time.  | Long  	|
+| `stopSequence1`       | `stop_sequence` of the first stop.      | Integer	|
+| `stopId1`             | `stop_id` of the first stop.            | String 	|
+| `stopName1`           | `stop_name` of the first stop.          | String 	|
+| `departureTime1`      | `departure_time` of the first stop.     | Time 	|
+| `csvRowNumber2`       | The row number of the second stop time. | Long 	|
+| `stopSequence2`       | `stop_sequence` of the second stop.     | Integer	|
+| `stopId2`             | `stop_id` of the second stop.           | String 	|
+| `stopName2`           | `stop_name` of the second stop.         | String 	|
+| `arrivalTime2`        | `arrival_time` of the second stop.      | Time 	|
+
+##### Speed thresholds
```

Comment:
@lionel-nj Is this table better off in RULES.md?

My impression from other comments is that NOTICES.md will go away when you can export the notice schemas.

Reply:
Indeed, this would be better in `RULES.md` in order to document the rule's logic.

Reply:
Sure, moved.